### PR TITLE
Feature/ch144723/add placekey module to snowflake

### PR DESCRIPTION
**Pull request by vdelacruzb**
Will also have to update doc in placekey, as now it uses h3 hexadecimal values for conversions.

### CODE REVIEW

**Comment by alex-graciano** [comment]
I see some conflicts in `tools/dev_environment.sh.sample`. @vdelacruzb could yo take a look to it?

**Comment by vdelacruzb** [comment]
@alex-graciano sure going to resolve them and rebase master.

File: placekey/sf/doc/REFERENCE.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
+## placekey
+
+[Placekey](https://www.placekey.io/faq) is a free and open universal standard identifier for any physical place, so that the data pertaining to those places can be shared across organizations easily. Since Placekey is based on H3, here we offer a way to transform to and from that index and delegate any extra functionality to H3 itself.
+
+You can learn more about Placekey on [their website](https://www.placekey.io/) or in the [Overview section](../../overview/spatial-indexes/#placekey) of this documentation.
+
+### H3_ASPLACEKEY
+
+{{% bannerNote type="code" %}}
+PLACEKEY.H3_ASPLACEKEY(h3index)
```
Comment (Jesus89) [inline placekey/sf/doc/REFERENCE.md:10]: I would use lowercase for the module name to be aligned with the BigQuery reference.

placekey.H3_ASPLACEKEY(h3index)

File: placekey/sf/doc/REFERENCE.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+* `h3index`: `STRING` H3 identifier.
+
+**Return type**
+
+`STRING`
+
+**Example**
+
+```sql
+SELECT SFCARTO.PLACEKEY.H3_ASPLACEKEY('847b59dffffffff');
```
Comment (Jesus89) [inline placekey/sf/doc/REFERENCE.md:26]: I would use lowercase for the project name and module name to be aligned with the BigQuery reference. Also we should use `"` for the strings in the examples too.

```sql
SELECT sfcarto.placekey.H3_ASPLACEKEY("847b59dffffffff");
```

**Review by Jesus89 — COMMENTED** [review]
Great job!

I have just added a couple of comments regarding the style of the reference in general.

File: placekey/sf/doc/REFERENCE.md (hunk 2 above)
Comment (vdelacruzb) [reply placekey/sf/doc/REFERENCE.md:26]: yep, I remember. Actually, I was planning to do a pass on all the modules to update the doc, accordingly. Will update this one and create a separate PR to sync docs, READMEs and some other little changes in the rest of the modules

**Review by Jesus89 — APPROVED** [review]
LGTM

**Pull request merged**